config/jobs/kubernetes/sig-network/dualstack-e2e.yaml

```diff
@@ -261,7 +261,64 @@ periodics:
     testgrid-tab-name: dualstack-azure-e2e-master-ipvs
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: "Dual-stack and Conformance e2e tests on a Kubernetes cluster provided by aks-engine (https://github.com/Azure/aks-engine) on Azure cloud in ipvs mode"
-
+- interval: 24h
+  name: ci-dualstack-azure-e2e-master-windows
+  decorate: true
+  decoration_config:
+    timeout: 5h
+  labels:
+    preset-service-account: "true"
+    preset-azure-cred: "true"
+    preset-dind-enabled: "true"
+    preset-azure-windows: "true"
+    preset-k8s-ssh: "true"
+    preset-windows-private-registry-cred: "true"
+    preset-windows-repo-list-master: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20210601-ea6aa4e-master
+      command:
+      - runner.sh
+      - kubetest
+      args:
+      # Generic e2e test args
+      - --test
+      - --up
+      - --down
+      - --build=quick
+      - --dump=$(ARTIFACTS)
+      # Azure-specific test args
+      - --provider=skeleton
+      - --deployment=aksengine
+      - --aksengine-agentpoolcount=2
+      - --aksengine-admin-username=azureuser
+      - --aksengine-admin-password=AdminPassw0rd
```

Comment:
Is this for real?😂

Reply:
It's a placeholder password, we don't actually use it

Reply:
yeah, I thought that it should be something like that, was only a joke :)